src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,75 @@
+import {useMemo, useState} from "react";
+import CloseIcon from '@mui/icons-material/Close';
+import CampaignIcon from '@mui/icons-material/Campaign';
+import './Banner.css';
+
+export default function Banner(props) {
+    const [show, setShow] = useState(true);
+    const bannerPlace = useMemo(() => {
+        return props.bannerPlace === 'bottom' ? {
+            bottom: 0,
+            position: 'fixed',
+            width: '100%'
+        } : {}
+    }, [props.bannerPlace])
+    if (window.localStorage.getItem('seen_banner')) {
+        return <></>
+    }
+    return (show ?
+        <div style={{...props.bannerStyle, ...bannerPlace}}
+             className="bg-indigo-600">
```

Comment:
Codacy has a fix for the issue: [Replace `·············className="bg-indigo-600">` with `······className="bg-indigo-600"`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
      className="bg-indigo-600"
```